### ISSUE TITLE
[2022 spring] ICCV 2021 RSLAD (20215437)

### ISSUE THREAD

**Issue by sky0701** [awesome-davian/awesome-reviews-kaist#482]
Thank you for awesome review. i have some basic questions on this paper.
1. What is clean sample in this paper? 
2. is training setting same in CIFAR-10 and CIFAR-100? (epoch, etc.)

Thank you

**Comment by monouns** [comment]
Thank you for awesome review! I'm also interested in attack robustness.
You explained <Problem Definition> and <Motivation>, so I understood why this paper is important.
However, with <Method> part, I'm hard to understand exactly what method is being used to improve attack robustness performance.
I think detailed model architecture image and more explanation will make more great review :)
Thank you! (20214418)

**Comment by stitsyuk** [comment]
Reviewer: Artyom Stitsyuk 20218256

Actually, I liked the overall structure and the content of the paper review. The idea is explained smoothly and the content is written in an easy-understandable way. The review is not long, but it is capacious so that it seems that all parts of the paper were delivered well. Nevertheless, there are some minor typos and moments that I suggest you to pay attention at:

•	In the “3. Method” section, the **total loss function formula is marked as (3) on the image** while it is the first formula in the review.
•	It was quiet complicated for me to understand **how does the supervision of student model by adversarially trained teacher model happens**. May be this part could be explained a bit deeper since in my opinion this is the essence of the whole paper. 
•	In the “4. Experiment & Result” section some **reference links seem to be wrong**. The review has 10 references, but in this block there are “ResNet-18 [19]”, “MobileNetV2 [37]”, “WideResNet34-10 [56]”, and “WideResNet-70-16 [16]”.
•	A **typo** in the “White-box robustness” subsection. It is written “Table 4 for CIFAR-100” while it should be Table 3.
•	While the quantitative result is provided in the review, I also would like to see the **qualitative results**. 

Thank you for your work.
